### PR TITLE
+faraday.0.5.3

### PR DESCRIPTION
**Pull request by seliopou**

### CODE REVIEW

**Comment by mseri** [comment]
```# File "lwt_unix/faraday_lwt_unix.mli", line 7, characters 5-24:
# Error: Unbound module Lwt_unix```

File: packages/faraday-lwt-unix/faraday-lwt-unix.0.3.0/opam [hunk 1]
```diff
@@ -16,7 +16,7 @@ depends: [
   "jbuilder" {build & >= "1.0+beta10"}
   "faraday" {>= "0.3.0" & < "0.5.0"}
   "faraday-lwt"
-  "lwt"
+  "lwt" {< 4.0.0}
```
Comment (mseri) [inline packages/faraday-lwt-unix/faraday-lwt-unix.0.3.0/opam]: Oops, missing quotes: it hsould be “4.0.0”

File: packages/faraday-lwt-unix/faraday-lwt-unix.0.5.0/opam [hunk 2]
```diff
@@ -15,7 +15,7 @@ build-test: [
 depends: [
   "jbuilder" {build & >= "1.0+beta10"}
   "faraday-lwt"
-  "lwt" {< "4.0.0"}
+  "lwt" {< ""4.0.0""}
```
Comment (hcarty) [inline packages/faraday-lwt-unix/faraday-lwt-unix.0.5.0/opam]: Extra quotes on `""4.0.0""`

File: packages/faraday-lwt-unix/faraday-lwt-unix.0.5.1/opam [hunk 3]
```diff
@@ -15,7 +15,7 @@ build-test: [
 depends: [
   "jbuilder" {build & >= "1.0+beta10"}
   "faraday-lwt"
-  "lwt" {< "4.0.0"}
+  "lwt" {< ""4.0.0""}
```
Comment (hcarty) [inline packages/faraday-lwt-unix/faraday-lwt-unix.0.5.1/opam]: Same extra quotes one `""4.0.0""`

**Comment by hcarty** [comment]
The Travis build is still failing for some reason, even though the datakit builds seem to be OK:
```
[ERROR] The compilation of faraday-lwt failed at "jbuilder build -p faraday-lwt
        -j 4".
#=== ERROR while installing faraday-lwt.0.5.3 =================================#
# opam-version 1.2.2
# os           darwin
# command      jbuilder build -p faraday-lwt -j 4
# path         /Users/travis/.opam/system/build/faraday-lwt.0.5.3
# compiler     system (4.06.1)
# exit-code    1
# env-file     /Users/travis/.opam/system/build/faraday-lwt.0.5.3/faraday-lwt-30534-4ec726.env
# stdout-file  /Users/travis/.opam/system/build/faraday-lwt.0.5.3/faraday-lwt-30534-4ec726.out
# stderr-file  /Users/travis/.opam/system/build/faraday-lwt.0.5.3/faraday-lwt-30534-4ec726.err
### stderr ###
# Error: The implementation lwt/faraday_lwt.ml
# [...]
#          val serialize :
#            Faraday.t ->
#            yield:(Faraday.t -> unit Lwt.t) ->
#            writev:(Faraday.bigstring Faraday.iovec list ->
#                    [ `Closed | `Ok of int ] Lwt.t) ->
#            unit Lwt.t
#        File "lwt/faraday_lwt.mli", line 4, characters 0-139:
#          Expected declaration
#        File "lwt/faraday_lwt.ml", line 3, characters 4-13: Actual declaration
```

Older `faraday-lwt*` packages look like they need Lwt version constraints as well.

**Comment by kit-ty-kate** [comment]
ping @seliopou

**Comment by seliopou** [comment]
This needs some jbuild changes, which will be included in the next release. Closing this in the mean time.

**Pull request closed without merging**